src/ripple/protocol/Serializer.h

```diff
@@ -298,7 +297,7 @@ class Serializer
     }
 
     // low-level VL length encode/decode functions
-    static Blob encodeVL (int length);
+    static int encodeVL (int length, std::array<std::uint8_t, 4>& result);
```

Comment:
I'm cool with not doing the privatizing audit.  It was definitely a scope creep kind of request.